pkgs/desktops/gnome-3/core/tracker-miners/default.nix

```diff
@@ -51,6 +84,18 @@ in stdenv.mkDerivation rec {
     };
```

Comment:
Could you replace the poppler thing with the following patch

```nix
    # fix poppler-glib include
    # https://bugzilla.gnome.org/show_bug.cgi?id=795573
    (fetchurl {
      url = https://bugzilla.gnome.org/attachment.cgi?id=371422;
      sha256 = "1rzwzrc7q73k42s1j1iw52chy10w6y3xksfrzg2l42nn9wk7n281";
    })
```

Reply:
Done!